database/db.go

```diff
@@ -59,16 +61,31 @@ func (d dbLogger) AfterQuery(q *pg.QueryEvent) {
 
 // New returns a new DB object which wraps
 // a connection to the database specified in config
-func New(config DBConfig) DB {
+func New(config DBConfig, logger *log.Logger) DB {
 	db := pg.Connect(&pg.Options{
 		Addr:     config.Address,
 		User:     config.User,
 		Database: config.Database,
 		Password: config.Password,
 	})
-	if os.Getenv("QUERY_DEBUG") == "ON" {
-		db.AddQueryHook(dbLogger{logger: log.New(os.Stdout, fmt.Sprintf("%s: %s: ", config.Address, config.Database), log.LstdFlags)})
+	if config.EnableLogging {
+		db.AddQueryHook(dbLogger{logger: logger})
+	}
+	return DB{
+		Client: db,
+		Logger: logger,
+	}
+}
+
+// Initialize starts up the database and returns the Close function used to gracefully shut it down.
```

Comment:
this comment needs to be updated to be accurate, there is no database started, only an attempt to run migrations(which as a consumer of this code I would like to know is going to happen when I call this code), which I don't think should happen as it prevents someone from using this who wants to do something non standard/consider the database initialized in a different way.. Please keep in mind when adding code to this module that the purpose of utils is to provide flexible general purpose code, not become the abstract class that imposes behavior on consumers.